### PR TITLE
Feat support pull action, call microflow and nanoflow

### PR DESCRIPTION
**Pull request by Andries-Smit**
Add support for various action on pull down:
* Refresh
* Call Nanoflow
* Call Microflow

Adding nanoflow support, sets the minimal version 7.13.1

### CODE REVIEW

File: src/PullToRefresh/widget/handlers/PullToRefresh.ts [hunk 1]
```diff
@@ -86,9 +86,9 @@ export class PullToRefresh {
 
     private update(nextState: State) {
         const { iconElement, textElement, pullToRefreshElement, classPrefix, iconRefreshing, iconArrow,
-            refreshText, pullToRefreshText, releaseToRefreshText, reloadDistance } = this.settings;
+                refreshText, pullToRefreshText, releaseToRefreshText, reloadDistance } = this.settings;
```
Comment (diego-antonelli) [inline src/PullToRefresh/widget/handlers/PullToRefresh.ts:89]: extra space

File: src/PullToRefresh/widget/PullToRefresh.xml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <widget id="PullToRefresh.widget.PullToRefresh" needsEntityContext="false"  offlineCapable="true" xmlns="http://www.mendix.com/widget/1.0/">
-    <name>Pull to refresh</name>
-    <description>Use touch on mobile enabling pull down to reload a page</description>
+    <name>Pull Action</name>
```
Comment (diego-antonelli) [inline src/PullToRefresh/widget/PullToRefresh.xml:3]: Not sure if we should change the name

File: src/PullToRefresh/widget/handlers/PullToRefresh.ts (hunk 1 above)
Comment (Andries-Smit) [reply src/PullToRefresh/widget/handlers/PullToRefresh.ts:89]: Space was created by eslint fix.

File: src/PullToRefresh/widget/PullToRefresh.xml (hunk 2 above)
Comment (Andries-Smit) [reply src/PullToRefresh/widget/PullToRefresh.xml:3]: Will revert, as the preview image a image anyway

**Pull request merged**